packages/extension-ui/src/components/View.tsx

```diff
@@ -49,6 +51,13 @@ export const BodyTheme = createGlobalStyle<ThemeProps>`
       display: none;
     }
   }
+
+  div#root {
+    height: calc(100vh - 2px);
+    margin: 0 auto;
+    max-width: 100%;
+    padding: 0;
+  }
```

Comment:
I think giving html and body 100% allows us to give every child 100% without worrying about +/- px :thinking:

Reply:
What are you suggesting exactly?

Reply:
```
html {
  height: 100%;
}
body {
  height: 100%;
}
div#root {
  height: 100%;
}

Reply:
Yeah, I left these styles _as is_, but I think we can do that 👍